neutron/templates/agents/network_agents.yaml

```diff
@@ -0,0 +1,7 @@
+{{- $context := . -}}
```

Comment:
This is not really needed. The inital scope of "." can be accessed in a range loop by using "$." instead.